### DISCUSSION TITLE
issue with firefly update to 2.03

### DISCUSSION THREAD

**Discussion by Kadett75** [iotaledger/firefly#7669] [Help & Support]
Hello
i did update of firefly and when update was done the total amount of iota on the wallet was 4000 less than before.
how can i restore and is this a known issue?
thanks

**Comment by begonaalvarezd** [comment]
Please run the balance finder until your balance is fully discovered, settings > advanced > balance finder. If you only need to search for the current wallet you can check the checkbox to speed up the search process

**Comment by Kadett75** [comment]
Thanks will try Am 27.10.2023 um 15:51 schrieb Begoña Álvarez de la Cruz ***@***.***>:﻿
Please run the balance finder until your balance is fully discovered, settings > advanced > balance finder. If you only need to search for the current wallet you can check the checkbox to speed up the search process

**Comment by Kadett75** [comment]
Hello Begona!THANKS A LOT!!!This exactly resolved my issue of missing funds.Now only history is missing, but that is not so critical Have a nice weekend Am 27.10.2023 um 22:36 schrieb MCM ***@***.***>:﻿Thanks will try Am 27.10.2023 um 15:51 schrieb Begoña Álvarez de la Cruz ***@***.***>:﻿
Please run the balance finder until your balance is fully discovered, settings > advanced > balance finder. If you only need to search for the current wallet you can check the checkbox to speed up the search process
